pallets/asset/src/lib.rs

```diff
@@ -185,6 +180,8 @@ pub trait Trait:
     /// Max length of the funding round name.
     type FundingRoundNameMaxLength: Get<usize>;
 
+    type AssetFn: AssetFnTrait<Self::Balance, Self::AccountId, Self::Origin>;
```

Comment:
This seems rather roundabout to me. I think it would be preferable to move the `pub trait Trait` to a commonly reachable location so that you can continue just having one type parameter `T: AssetTrait` for all the benches and not have to define two versions of `make_asset`. The associated type should then also be unnecessary, and instead you'll use:
```rust
where_clause {  where T: AssetTrait }
```
in benchmarks.

Reply:
@Centril Could you please make a PR with that suggestion? I did not see how to do that.

Reply:
Sure, I'll give it a try :)